admin/views/tabs/metas/paper-content/post-type-content.php

```diff
@@ -67,7 +75,14 @@
 /**
  * Allow adding a custom checkboxes to the admin meta page - Post Types tab.
  *
+ * @deprecated 16.2 Use the {@see 'Yoast\WP\SEO\admin_post_types_meta_before_archive'} action instead.
+ *
  * @api  WPSEO_Admin_Pages  $yform  The WPSEO_Admin_Pages object
```

Comment:
I think we can fix these two? So we have less things to convert to `@param` when and if that will be enforced...